src/components/Cluster/ClusterDetail/ClusterDetailView.js

```diff
@@ -78,27 +78,38 @@ class ClusterDetailView extends React.Component {
     );
   };
 
-  loadDetails = () => {
-    const { cluster, clusterId, organizationId, dispatch } = this.props;
+  // It is not in your orgs or it has been deleted.
+  doesNotExist = (isDeleted) => {
+    const { clusterId, organizationId, dispatch } = this.props;
 
-    if (typeof cluster === 'undefined' || cluster.delete_date) {
-      const organizationDetailPath = RoutePath.createUsablePath(
-        OrganizationsRoutes.Detail,
-        {
-          orgId: organizationId,
-        }
-      );
+    const organizationDetailPath = RoutePath.createUsablePath(
+      OrganizationsRoutes.Detail,
+      {
+        orgId: organizationId,
+      }
+    );
```

Comment:
Do we really want to redirect the user to the organization view?  Why not redirect him to the home view?

Reply:
I think the home view would probably be nicer yeah.